django/http/response.py

```diff
@@ -366,12 +366,31 @@ class HttpResponse(HttpResponseBase):
     """
 
     streaming = False
+    non_picklable_attrs = frozenset(
+        [
+            "resolver_match",
+            # Non-pickleable attributes added by test clients.
+            "asgi_request",
```

Comment:
I don't see any failure after removing `asgi_request` (see https://github.com/django/django/pull/14728#discussion_r702723854). Please add a test for the async-client.

Reply:
Ok 👍🏻

Reply:
Had a small doubt that should I use the predefined `async_get_view` or make one of my own?